packages/components/src/components/chip/chip.tsx

```diff
@@ -8,35 +8,45 @@ import { JSX } from '@stencil/core/internal';
 })
 export class Chip implements ComponentInterface {
 
-  /**
-   * The icon source.
-   *
-   * @type {string}
-   */
-  @Prop() icon?: string;
-
-  /**
-   * The chip content that is displayed in the component.
-   *
-   * @type {string}
-   */
-
-  @Prop() content: string;
-
-  /**
-   * Checks if the chip is active and applies different styling to the component.
-   *
-   * @type {boolean}
-   */
-
-  @Prop() active: boolean = false;
-
-  render(): JSX.Element {
-      return (
-          <div class={`chip ${this.active ? 'chip--active' : ''}`}>
-              {this.icon && <img class="chip__icon" src={this.icon}></img>}
-              <div class="chip__content">{this.content}</div>
-          </div>
-      );
-  }
+    /**
+     * The icon source.
+     *
+     * @type {string}
+     */
+    @Prop() icon?: string;
+
+    /**
+     * The chip content that is displayed in the component.
+     *
+     * @type {string}
+     */
+
+    @Prop() content: string;
+
+    /**
+     * The primary color of the chip.
+     *
+     * @type {string}
+     */
+    @Prop() primaryColor?: string;
```

Comment:
It's used for background color only. Should we call it that instead?
And should we write what the default value is?

Reply:
Yeah, good idea. Renamed it to `backgroundColor` instead, and added default color as well